### PR TITLE
hb-07

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

**Review by dariiabutorina — APPROVED** [review]
Well done! 
Let's override toString() methods for our models for them to display correctly in the exceptions messages

File: src/main/java/cinema/dao/impl/OrderDaoImpl.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getOrdersByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Order> orders = session.createQuery("FROM Order o "
+                    + "LEFT JOIN FETCH o.tickets "
+                    + "JOIN FETCH o.user "
```
Comment (dmytro-hryhoruk) [inline src/main/java/cinema/dao/impl/OrderDaoImpl.java]: do we need to fetch User since you are using ManyToOne for this field?

Comment (julia-shvabovich) [reply src/main/java/cinema/dao/impl/OrderDaoImpl.java]: done

File: src/main/java/cinema/dao/impl/OrderDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            }
+        }
+    }
+
+    @Override
+    public List<Order> getOrdersByUser(User user) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Order> orders = session.createQuery("FROM Order o "
+                    + "LEFT JOIN FETCH o.tickets "
+                    + "WHERE o.user = :user", Order.class);
```
Comment (dmytro-hryhoruk) [inline src/main/java/cinema/dao/impl/OrderDaoImpl.java]: let's use distinct keyword in order to get rid of duplications if our order has more than one ticket

Comment (julia-shvabovich) [reply src/main/java/cinema/dao/impl/OrderDaoImpl.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**